File: Backend/Models/comment.go
package models

type StoryComment struct {
	CommentId int `json:"commentId"`
	StoryId int `json:"storyId"`
	ParentCommentId int `json:"parentCommentId"`
	Content int `json:"content"`
	DateCreated string `json:"dateCreated"`
	UpdatedAt string `json:"updatedAt"`,
	Username string `json:"username"`,
	Votes string `json:"votes"`
} 

File: Frontend/src/app/types/comment.ts
import { CommentVote } from "./vote";

export type StoryComment = {
	commentId: number;
	storyId: number;
	userId: number;
	parentCommentId: number;
	content: string;
	votes: Array<CommentVote>;
	dateCreated: string;
	updatedAt: string;
	username: string;
}